### DISCUSSION TITLE
Welcome to OpenWorm Discussions!

### DISCUSSION THREAD

**Discussion by slarson** [openworm/OpenWorm#311] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

   To start off with, please have a look at [OpenWorm's documentation](http://docs.openworm.org) and search for any questions you'd like the answers to.
  
   To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saif-q** [comment]
Hey I want to ask some questions but my most important one could I use some of technology in a future video game idea?

**Comment by saif-q** [comment]
Second question how long will this worm take to be completely coded

**Comment by saif-q** [comment]
Third question can I join your team

**Comment by saif-q** [comment]
And last question what is your team going to do after this project is over

**Comment by saif-q** [comment]
Just some curiosity how much do your team get paid and what is the reasoning for this

**Comment by sherlocklecter7201** [comment]
![image](https://github.com/openworm/OpenWorm/assets/168358371/646a37a8-2a75-46d6-a907-d82b369e13e2)
Hello I have this error message popping up before completing the execution and I cant seem to find a solution, could you tell me what I am doing wrong?

**Comment by shyanhaque** [comment]
Hi is there any place where I can find healthy and injured image datasets? I need them to train an lstm model for posture detection. Also it is very urgent that i find these datasets within two days.
